node/integrationtest/yarn.js

```diff
@@ -124028,7 +124028,7 @@ module.exports = function (iconv) {
     // Node authors rewrote Buffer internals to make it compatible with
     // Uint8Array and we cannot patch key functions since then.
     // Note: this does use older Buffer API on a purpose
-    iconv.supportsNodeEncodingsExtension = !(Buffer.from || new Buffer(0) instanceof Uint8Array);
+    iconv.supportsNodeEncodingsExtension = !(Buffer.from || Buffer.from(0) instanceof Uint8Array);
```

Comment:
This will throw.

Reply:
I now replaced `Buffer.from(<integer>)` with `Buffer.alloc(<integer>)`.